mainapp/tests.py

```diff
@@ -209,3 +209,36 @@ def test_user_rated_book_valid(self):
         self.assertEquals(rating.bookrating, valid_bookrating)
         self.assertEquals(rating.user, self.user)
         self.client.logout()
+
+
+class RatedBooksTestCase(TestCase):
+    '''
+    Read books view test case
+    '''
```

Comment:
```suggestion
    """Already Read Books View Test Case"""
```

Reply:
Would suggest to follow this for every function, we are currently updating our docs #36